E0329 14:14:45.007495 1766592 portforward.go:404] "Unhandled Error" err="error copying from local connection to remote stream: writeto tcp4 127.0.0.1:36185->127.0.0.1:49728: read tcp4 127.0.0.1:36185->127.0.0.1:49728: read: connection reset by peer" logger="UnhandledError"
panic in flow watcher: test panic
panic: test panic [recovered]
	panic: test panic

goroutine 99 [running]:
github.com/doucol/clyde/internal/whisker.WatchFlows.func2()
	/home/doug/dev/clyde/internal/whisker/whisker.go:63 +0x95
panic({0x1a0a1e0?, 0x208f800?})
	/usr/local/go/src/runtime/panic.go:785 +0x132
github.com/doucol/clyde/internal/whisker.WatchFlows.func1({0xc0013d2006, 0x36e})
	/home/doug/dev/clyde/internal/whisker/whisker.go:43 +0xf8
github.com/doucol/clyde/internal/catcher.(*DataCatcher).consumeSSEStream(0xc000402000, {0x20b5048, 0xc0000b5440}, {0xc000412690?, 0x2?}, 0xc000406e70)
	/home/doug/dev/clyde/internal/catcher/catcher.go:196 +0x503
github.com/doucol/clyde/internal/catcher.(*DataCatcher).CatchDataFromSSEStream.func3()
	/home/doug/dev/clyde/internal/catcher/catcher.go:135 +0x22a
created by github.com/doucol/clyde/internal/catcher.(*DataCatcher).CatchDataFromSSEStream in goroutine 26
	/home/doug/dev/clyde/internal/catcher/catcher.go:127 +0x6d3
